.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: release
+
+on:
+  push:
+    branches:
+      - [main]
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
```

Comment:
You don't usually have to set this because the default is `${{ github.token }}`

Reply:
No, you cannot. The default GitHub token doesn't has sufficient permissions needed to create releases.